Added stream support.

deepjyoti30 opened the pull request:
Removed caching for the time being. It directly streams now using mpd and mpc

NISH1001 commented:
Hi,
Thanks for the PR. I see you have done direct streaming. But I have a concern here.
Have you removed the whole cache module? I think direct streaming could have been done using a new module. I will merge it once I find it "not breaking" the existing features.

deepjyoti30 commented:
For now, i have removed complete caching. I think we can add it later by making it download the song in the background and saving to cache directory.
If you check the files, i have removed cache.py for now.

deepjyoti30 commented:
Completely agree about that, but keeping the cache file right now would be completely useless. Since mod directly streams, i am just providing the url to it.
If you want, take a look at the forked repo once and tell me if you still need the cache file.
Later though we can add caching in a better way.
We can make mpd play while it downloads in the background using another thread.
That way it will be better, but as of now, i dont think we need that.

NISH1001 commented:
I am testing checking out the changes. I will get back to this thread once I am able to run it. Thanks. We can make this more like a music assistant for Linux if time favours.
